tensorflow/python/compiler/tensorrt/utils.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
2020. I think I will approve this and fix it in my local, to speed up the process to merge it.

Reply:
Fixed